Tasks/MavenV2/maventask.ts

```diff
@@ -336,9 +341,11 @@ function publishJUnitTestResults(testResultsFiles: string) {
         tl.debug('Pattern found in testResultsFiles parameter');
         var buildFolder = tl.getVariable('System.DefaultWorkingDirectory');
         tl.debug(`buildFolder=${buildFolder}`);
-        matchingJUnitResultFiles = tl.findMatch(buildFolder, testResultsFiles, null, {
-            matchBase: true
-        });
+        matchingJUnitResultFiles = tl.findMatch(buildFolder, testResultsFiles, {
+            allowBrokenSymbolicLinks: true,
```

Comment:
Would this be a breaking changes for user who already have some broken symlinks matching passed patterns?

Reply:
If users' test tries to read data from broken symlink, test will be failed and task also will be failed too.